### PR TITLE
Icons: Update SVGs

### LINKED ISSUE #288: Make icons larger
To match the changes for GNOME 41.

[Source](https://blogs.gnome.org/alexm/2021/09/10/cleaning-up-header-bars/)

### PR DESCRIPTION
**Pull request by oscfdezdz**
Fixes #288

### CODE REVIEW

File: theme/icons/tab-new-symbolic.svg [hunk 1]
```diff
@@ -1,6 +1,8 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="16.009" height="16">
-    <g fill="context-fill" stroke="context-stroke" fill-opacity="context-fill-opacity" stroke-opacity="context-stroke-opacity">
-        <path d="M4.003 1s-.709-.014-1.447.355C1.817 1.725 1.003 2.667 1.003 4v9h-1l-.004 2h.004l3 .01V15h13v-2h-1V4s.014-.709-.355-1.447C14.278 1.814 13.336 1 12.003 1zm0 2h8c.667 0 .725.186.855.447.131.262.145.553.145.553v9h-10V4c0-.667.186-.725.447-.855.262-.131.553-.145.553-.145z" style="line-height:normal;font-variant-ligatures:normal;font-variant-position:normal;font-variant-caps:normal;font-variant-numeric:normal;font-variant-alternates:normal;font-feature-settings:normal;text-indent:0;text-align:start;text-decoration-line:none;text-decoration-style:solid;text-decoration-color:#000;text-transform:none;text-orientation:mixed;shape-padding:0;isolation:auto;mix-blend-mode:normal" color="#000" font-weight="400" font-family="sans-serif" white-space="normal" overflow="visible"/>
-        <path d="M7.009 5v2h-2v2h2v2h2V9h2V7h-2V5z"/>
+<?xml version="1.0" encoding="UTF-8"?>
+<svg height="16px" viewBox="0 0 16 16" width="16px" xmlns="http://www.w3.org/2000/svg">
+    <g fill="#2e3436">
```
Comment (rafaelmardojai) [inline theme/icons/tab-new-symbolic.svg:3]:
```suggestion
    <g fill="context-fill" stroke="context-stroke" fill-opacity="context-fill-opacity" stroke-opacity="context-stroke-opacity">
```
Please keep these custom properties for `fill`, `fill-opacity`, etc. This allow us to recolor the icons with CSS.

**Review by rafaelmardojai — CHANGES_REQUESTED** [review]

**Comment by oscfdezdz** [comment]
Should I update all SVGs in this PR? I'm already updating them locally.

**Comment by rafaelmardojai** [comment]
> Should I update all SVGs in this PR? I'm already updating them locally.

Yeah, better.

**Review by rafaelmardojai — APPROVED** [review]
Looks good! Thanks for working on this, really appreciated.

**Pull request merged**